helix-term/src/commands/typed.rs

```diff
@@ -2148,6 +2148,115 @@ fn sort_impl(cx: &mut compositor::Context, reverse: bool) -> anyhow::Result<()>
     Ok(())
 }
 
+fn index(cx: &mut compositor::Context, args: Args, event: PromptEvent) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    let step = if let Some(arg) = args.first() {
+        arg.parse()
+            .ok()
+            .filter(|&step| step != 0)
+            .context("Step must be valid unsigned integer and greater than zero")?
```

Comment:
```suggestion
            .context("Step must be a positive number greater than zero")?
```
Since not all Helix users are programmers